projects/observability/src/shared/components/entity-renderer/entity-renderer.component.scss

```diff
@@ -2,11 +2,13 @@
 @import 'font';
 
 .ht-entity-renderer {
+  @include body-1-regular();
```

Comment:
Why declare the font if we're also trying to inherit it?

Reply:
for the defaults that are not overridden.

Reply:
Is that anything beyond color? If it's not, can we expose a variable like default-body-color (and update the font mixins to something like)
```scss
$default-body-color:  $gray-9;
@mixin body-1-regular($color: $default-body-color) {
```
and this to just declare the color? I think it's confusing and prone to breaking to use a mixin for one property, and override the others intentionally.

Reply:
Anand is revamping the views. Dropping this PR.